client/components/product-expiration/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+interface Moment {
+	isValid: () => boolean;
+	format: ( arg0: string ) => string;
+	diff: ( arg0: Date ) => number;
+}
+
+export interface Props {
+	expiryDateMoment: Moment;
+	purchaseDateMoment: Moment;
+	isRefundable: boolean;
+	dateFormat: string;
+	translate: ( arg0: string, arg1: object ) => string;
```

Comment:
I think `translate` supports other syntaxes too, maybe we want to keep it simple. Or use `LocalizeProps` from `i18n-calypso`?

Reply:
So I tried to import the correct types from `i18n-calypso` and nothing went 💥 yet? So I'm tentatively calling this resolved by f3dac8282bf5ea70665c00f21aafe24a9a5c08be